src/Essentials/src/Permissions/Permissions.android.cs

```diff
@@ -521,14 +521,75 @@ public partial class Speech : BasePlatformPermission
 
 		public partial class StorageRead : BasePlatformPermission
 		{
-			public override (string androidPermission, bool isRuntime)[] RequiredPermissions =>
-				new (string, bool)[] { (Manifest.Permission.ReadExternalStorage, true) };
+			public override (string androidPermission, bool isRuntime)[] RequiredPermissions
+			{
+				get
+				{
+					if (!OperatingSystem.IsAndroidVersionAtLeast(33))
```

Comment:
Is this the correct check? Should it be `OperatingSystem.IsAndroidAtLeast(33)` basically without the `!` part?

If this is the case perhaps a private property to wrap this check might be nice? Something like:

```csharp
private bool DoesPermissionExist => OperatingSystem.IsAndroidAtLeast(33);
```

Reply:
Oops you're absolutely right, got the paths wrong.

Yeah the `!OperatingSystem.IsAndroidVersionAtLeast(33)` is very hard to read. We might need to add a `OperatingSystem.IsAndroidVersionLowerThan(33)` which under the hood does the same but is much more readable 😄